datastore/prometheus/views.py

```diff
@@ -85,13 +99,74 @@ def _total_num_sources_in_last_run(self):
             NUM_PROBLEM_SOURCES_IN_LAST_RUN.set(-1)
             NUM_OK_SOURCES_IN_LAST_RUN.set(0)
 
+    def _num_current_grants_with_beneficiary_location_geocode_without_lookup(self):
+        current_grants = db.Latest.objects.get(series=db.Latest.CURRENT).grant_set
+
```

Comment:
Didn't someone add a convince function this ? ;) 

https://github.com/ThreeSixtyGiving/datastore/blob/master/datastore/db/models.py#L22

Reply:
huh.... oh yeah! :D